packages/veritone-react-common/src/components/share-components/AlertDialog/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { Component } from 'react';
+import { any, bool, string, func, oneOfType} from 'prop-types';
+
+import Button from 'material-ui/Button';
+import Dialog, {
+  DialogActions,
+  DialogTitle,
+  DialogContent,
+  DialogContentText,
+} from 'material-ui/Dialog';
+
+import withMuiThemeProvider from 'helpers/withMuiThemeProvider';
+
+const labelId = 'veritone-alert-dialog-title';
+const descriptionId = 'veritone-alert-dialog-description';
+
+@withMuiThemeProvider
+export default class AlertDialog extends Component {
+  static propTypes = {
+    open: bool,
+    title: string,
+    content: string,
+    fullScreen:bool,
+    cancelButtonLabel: string,
+    approveButtonLabel: string,
+    onCancel: func,
+    onApprove: func.isRequired,
+    cancelValue: oneOfType([any]),
+    approveValue: oneOfType([any])
+  }
+
+  static defaultProps = {
+    open: false,
+    title: '',
+    content: '',
+    fullScreen: false,
+    cancelValue: 'cancel',
+    approveValue: 'approve',
+    cancelButtonLabel: 'Cancel',
+    approveButtonLabel: 'Continue'
+  }
+
+  handleCancel = (event) => {
+    this.props.onCancel && this.props.onCancel(this.props.cancelValue);
```

Comment:
fyi you're already checking if onCancel exists via the ternary in the render function, this check looks redundant